multichain-testing/tools/wallet-store-reflect.ts

```diff
@@ -0,0 +1,127 @@
+import type { SigningSmartWalletKit } from '@agoric/client-utils';
+import { retryUntilCondition, type RetryOptions } from '@agoric/client-utils';
+import type { UpdateRecord } from '@agoric/smart-wallet/src/smartWallet.js';
+import type { EMethods } from '@agoric/vow/src/E.js';
+
+export const walletUpdates = (
```

Comment:
I'm uncomfortable with more wallet client API that isn't in client-utils but I won't block

Reply:
I aim to revisit this, but I haven't managed in a couple weeks, and not having the `ymax-tool` stuff is getting painful, so @LuqiPan and I agreed to land this more or less as is, for now.